src/platform/plugins/shared/discover/public/plugin_imports/embeddable_services.ts

```diff
@@ -12,3 +12,4 @@ export { getSearchEmbeddableFactory } from '../embeddable/get_search_embeddable_
 export { getLegacyLogStreamEmbeddableFactory } from '../embeddable/get_legacy_log_stream_embeddable_factory';
 export { searchEmbeddableTransforms } from '../../common/embeddable';
 export { addControlsFromSavedSession } from '../embeddable/utils/add_controls_from_saved_session';
+export { SAVED_OBJECT_REF_NAME } from '@kbn/presentation-publishing';
```

Comment:
Previously we were dynamically importing all of `@kbn/presentation-publishing` in `plugin.tsx` just for a single const. This bundles it in with our other embeddable services and supports tree shaking.